lib/r10k/git/rjgit.rb

```
@@ -60,6 +60,13 @@ def rev_parse(ver, opts = {})
         return commit.id
       end
 
+      def blob_at(ver, path, opts = {})
+        repo = ::RJGit::Repo.new(opts[:git_dir], is_bare: true)
+        blob = repo.blob(path, ver)
+
+        return blob.data
```

Do we want to raise an Error on failure as we're doing in the shellgit version of this?